client/my-sites/subscribers/components/subscriber-totals/index.tsx

```diff
@@ -75,28 +94,28 @@ const SubscriberTotals: React.FC< SubscriberTotalsProps > = ( {
 				<>
 					<span className="subscriber-totals__filtered-count">
 						{ searchTerm
-							? translate( '%(count)d matching result', '%(count)d matching results', {
-									count: filteredCount,
-									args: { count: filteredCount },
+							? translate( '%(count)s matching result', '%(count)s matching results', {
+									count: filteredCount || 2,
```

Comment:
I'm seeing "0 matched result" in a non-EN language, so I added this extra logic. But maybe it's just the missing translation?